### PR TITLE
Kubernetes Artifacts for OB Deployment Pattern 2 for UK specification

### PR DESCRIPTION
**Pull request by kaumini**

### CODE REVIEW

File: ob-pattern-2/templates/obam/wso2ob-pattern-2-obam-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wso2ob-pattern-2-obam-deployment
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: {{ default 1 .Values.wso2.deployment.wso2obam.replicas }}
```
Comment (chirangaalwis) [inline ob-pattern-2/templates/obam/wso2ob-pattern-2-obam-deployment.yaml]: @kaumini @kdanW can you please remove the `default` values being set at the resource definitions?

We can have the default values set at one location (i.e. only at the `values.yaml` file).

File: ob-pattern-2/templates/obbi-dashboard/wso2ob-pattern-2-obbi-dashboard-service.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (chirangaalwis) [inline ob-pattern-2/templates/obbi-dashboard/wso2ob-pattern-2-obbi-dashboard-service.yaml]: @kaumini update the license header year.

File: ob-pattern-2/templates/obbi-dashboard/wso2ob-pattern-2-obbi-dashboard-service.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    protocol: TCP
+  - name: rest-api-port-5
+    port: 7613
+    targetPort: 7613
+    protocol: TCP
+  - name: rest-api-port-6
+    port: 8006
+    targetPort: 8006
+    protocol: TCP
+
```
Comment (chirangaalwis) [inline ob-pattern-2/templates/obbi-dashboard/wso2ob-pattern-2-obbi-dashboard-service.yaml]: @kaumini remove the unnecessary, additional EOF lines. Applies to some other files as well.

File: ob-pattern-2/templates/obbi-worker/wso2ob-pattern-2-obbi-worker-service.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (chirangaalwis) [inline ob-pattern-2/templates/obbi-worker/wso2ob-pattern-2-obbi-worker-service.yaml]: @kaumini update the license header year.

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

File: ob-pattern-2/templates/obam/wso2ob-pattern-2-obam-deployment.yaml (hunk 1 above)
Comment (kaumini) [reply ob-pattern-2/templates/obam/wso2ob-pattern-2-obam-deployment.yaml]: Fixed with 68829a83552a35f361155eb8c7c65c46e7fe2711

File: ob-pattern-2/templates/obbi-dashboard/wso2ob-pattern-2-obbi-dashboard-service.yaml (hunk 2 above)
Comment (kaumini) [reply ob-pattern-2/templates/obbi-dashboard/wso2ob-pattern-2-obbi-dashboard-service.yaml]: Fixed with edf1bb86b4ea94c37158885b59e738d865e2347a

File: ob-pattern-2/templates/obbi-dashboard/wso2ob-pattern-2-obbi-dashboard-service.yaml (hunk 3 above)
Comment (kaumini) [reply ob-pattern-2/templates/obbi-dashboard/wso2ob-pattern-2-obbi-dashboard-service.yaml]: Fixed with edf1bb86b4ea94c37158885b59e738d865e2347a

File: ob-pattern-2/templates/obbi-worker/wso2ob-pattern-2-obbi-worker-service.yaml (hunk 4 above)
Comment (kaumini) [reply ob-pattern-2/templates/obbi-worker/wso2ob-pattern-2-obbi-worker-service.yaml]: Fixed with edf1bb86b4ea94c37158885b59e738d865e2347a

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Changelog
+All notable changes to this project 1.4.x per each release will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+
+## [v1.4.0.1] - 2019-11-28
+
+### Added
+- Introduce Helm charts for WSO2 Open Banking pattern 1
```
Comment (chirangaalwis) [inline CHANGELOG.md]: @kaumini has to be `...Helm Chart...`.

File: ob-pattern-2/values.yaml [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+          # The maximum amount of memory that should be allocated for a Pod
+          memory: "2Gi"
+          # The maximum amount of CPU that should be allocated for a Pod
+          cpu: "2000m"
+      # Refer to the Kubernetes documentation on updating images (https://kubernetes.io/docs/concepts/containers/images/#updating-images)
+      imagePullPolicy: Always
+
+kubernetes:
+  # Name of Kubernetes service account
+  svcaccount: "wso2svc-account"
```
Comment (chirangaalwis) [inline ob-pattern-2/values.yaml]: @kaumini shall we change this parameter name to `serviceAccount`?

Comment (chirangaalwis) [reply ob-pattern-2/values.yaml]: @kdanW applies to pattern 1 as well.

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (kaumini) [reply CHANGELOG.md]: Fixed with 859a08f3bb7f9b8ec314bc9b4f1a9704e0b1324b

File: ob-pattern-2/values.yaml (hunk 6 above)
Comment (kaumini) [reply ob-pattern-2/values.yaml]: Fixed with 84268724a46bc3dc3b34b28b33f212be7087fb00

**Review by chirangaalwis — APPROVED** [review]

**Pull request merged**